container/bin/entrypoint.py

```diff
@@ -0,0 +1,75 @@
+#!/venv/bin/python3
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import argparse
+import json
+import os
+import subprocess
+
+from ovirt_imageio.client import info
+from ovirt_imageio.client._api import _tmp_dir
+from ovirt_imageio.client._app import setup_signals
```

Comment:
I just wanted to avoid the traceback in the output:
```
podman run \
        --interactive \
        --tty \
        --rm \
        --publish 8080:80 \
        --volume /var/tmp/images:/var/tmp:Z \
        localhost/ovirt-imageio:latest /var/tmp/disk.raw
2022-10-26 11:19:14,860 INFO    (MainThread) [server] Starting (hostname=2897d4c9e137 pid=12, version=2.4.7)
2022-10-26 11:19:14,873 INFO    (MainThread) [services] remote.service listening on ('::', 80)
2022-10-26 11:19:14,873 INFO    (MainThread) [server] Initial ticket: /ticket.json
2022-10-26 11:19:14,874 INFO    (MainThread) [server] Ready for requests
^C2022-10-26 11:19:16,785 INFO    (MainThread) [server] Received signal 2, shutting down
2022-10-26 11:19:16,816 INFO    (MainThread) [server] Received signal 15, shutting down
Traceback (most recent call last):
  File "/bin/entrypoint.py", line 76, in <module>
    main()
  File "/bin/entrypoint.py", line 67, in main
    server.wait()
  File "/venv/lib/python3.10/site-packages/ovirt_imageio/_internal/qemu_nbd.py", line 226, in wait
    return self.proc.wait(timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1207, in wait
    return self._wait(timeout=timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1941, in _wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.10/subprocess.py", line 1899, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
```